OneCryptoPkg/OneCryptoLoaders/PeCoffLib.c

```diff
@@ -0,0 +1,205 @@
+/** @file
```

Comment:
Do you actually need a custom copy of this? Can we not use BasePeCoffLib?

Reply:
This isn't BasePeCoffLib - This is an extension of the functions of PeCoffLib. I changed the name. I'm not sure if I want to extend PeCoffLib right now to add functionality that I'm not sure should be widely public.